src/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -148,7 +148,14 @@ public QuicStream CreateUnidirectionalStream()
         {
             if (_mock)
             {
-                return new QuicStream(this, false);
+                int streamId;
+                lock (_syncObject)
+                {
+                    streamId = _nextOutboundUnidirectionalStream;
+                    _nextOutboundUnidirectionalStream += 4;
+                }
+
+                return new QuicStream(this, streamId, false);
```

Comment:
```suggestion
                return new QuicStream(this, streamId, canRead: false);
```

Reply:
> In this prototype, you are allowing QuicStreams to be new'd up, correct?

No, you can only create a QuicStream by calling QuicConnection.Create[Uni|Bi]DirectionalStream.